src/jquery.mockjax.js

```diff
@@ -269,6 +269,7 @@
 				url: mockHandler.proxy,
 				type: mockHandler.proxyType,
 				data: mockHandler.data,
+				async: false,
```

Comment:
Yes, I think that should solve your issue as well as not imposing synchronicity when not needed, correct?

See if that works for you, and if so, please be sure to write a test for it. Then we can get it into the project!